rfcs/2020-12-21-5664-trl-cli-distribution.md

```diff
@@ -0,0 +1,90 @@
+# RFC 5664 - 2020-12-21 - Distribution of the `trl` CLI tool
+
+The `trl` CLI tool for the Timber Remap Language is an extremely useful accompaniment to the
+language, but at the moment you can only use it inside of the Vector repo using `cargo run`. This
+RFC proposes distributing the `trl` tool via the same channels that we currently distribute Vector
+itself.
+
+## Scope
+
+This RFC only covers distribution of the `trl` tool as well as some related issues. It does *not*
+cover TRL syntax, functions, semantics, and the like.
+
+## Motivation
+
+TRL is likely to be widely used and a crucial part of Vector. Although TRL is relatively easy to use
+in conjunction with real Vector instances and topologies, e.g. via unit tests, the `trl` CLI tool
+allows for much speedier trial and error and thus will be an essential part of development flows
+for Vector.
+
+Asking users to run the CLI tool by cloning the Vector repo, installing Rust/Cargo, and running
+the command manually could be a significant impediment to TRL adoption.
+
+## Internal Proposal
+
+### General build setup
+
+Our build setup—`Makefile`, scripts, etc.—would need to be revised pretty significantly, as what we
```

Comment:
We could put the changes in a new `Makefile` in `lib/remap-cli`. I think we may want to make that an external repository later once things stabilize.